DittoHealth/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+<!--    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>-->
+
+
+    <application>
+        <provider
```

Comment:
Maybe not necessary, perhaps a copy/paste leftover

Reply:
Yeah,
I copy/paste one of the modules and I did not pay attention to the manifest. 
My bad.